### PR TITLE
Animation index

### PR DESCRIPTION
**Pull request by czakdesign**
_Revised index page  based on initial review comments
_pushed updated images
_removed unused images
_additional files are the image files

This is a  review for the index.md file.
Note:  comments from original PR were captured - but not all comments are incorporated in this PR.

@sfshaza @InMatrix -

### CODE REVIEW

File: animations/index.md [hunk 1]
```diff
@@ -1,159 +1,95 @@
...
-## Other resources
-
-Learn more about Flutter animations at the following links:
+## Flutter animation widgets
+Using the built-in Flutter animation widgets means you don’t have to write sophisticated drawing code. Flutter animation widgets include definitions for standard motion effects such as *fade out* or *slide transition* so most of the code required to draw each animation frame is already written for you.  
 
-* [Animations: Technical Overview](/animations/overview.html)<br>
-A look at some of the major classes in the animations library,
-and Flutter's animation architecture.
+Flutter animations widgets are *implicit*, *transition*, or *explicit* animations. Implicit and transition animations have most of the animation code already included so the level of customization is moderate. If you’re starting out in animations, or you want to quickly add an animation to your app, try out the implicit or transition animation widgets.
```
Comment (InMatrix) [inline animations/index.md:17]: Suggestion: Flutter animations widgets are *implicit*, *transition*, or *explicit* ~~animations~~.

File: animations/index.md [hunk 2]
```diff
@@ -1,159 +1,95 @@
...
-classes.
-
-### Staggered animation
+__________________________________  
 
-Animations that are broken into smaller motions, where some of the motion is delayed.
-The smaller animations may be sequential, or may partially or completely overlap.
-
-* [Staggered Animations](/animations/staggered-animations/) <img src="/images/ic_new_releases_black_24px.svg" alt="this doc is new!"> NEW<br>
+Well-designed animations makes a UI more intuitive, contributes to the sophisticated look and feel of a polished app, and improves the user experience. Flutter’s animation framework makes it easy to implement basic and advanced animation types—so whether you are new to animations or an animation wizard, Flutter provides the platform to bring your creative vision to life.
```
Comment (InMatrix) [inline animations/index.md:12]: This paragraph is not very necessary. Developers who come to this page probably don't need to be convinced that adding animations to their apps is a good idea. Then, the second sentence is sort of repeated in the "Flutter animation widgets" section.

File: animations/index.md [hunk 3]
```diff
@@ -1,159 +1,95 @@
...
 
-<hr>
 
-If there is specific animation documentation you'd like to see, please
-[file an issue](https://github.com/flutter/flutter/issues).
+### Implicit animation widgets
 
+ *Implicit animations* are animations that are already programmed or styled within pre-existing constraints—all you need to do is add the animated object, and set some basic parameters such as the duration. You don’t need to write the animation code to add listeners or tickers for example, and you don't need to tell the app when to start—the animation begins automatically when a property value changes. The `AnimatedOpacity` and `AnimatedIcon` widgets are two examples of implicit animations.
+ <br>
+ For more information including a list of implicit widgets, see [Implicit animations](/animations/implicit_widgets).
```
Comment (InMatrix) [inline animations/index.md:29]: I'd love to have some visual examples right here. We can reuse the same visuals in sub-pages. This index page is most important in engaging users. The user needs to quickly understand if Flutter can support the kind of motion design they have in mind by scanning this page.

File: animations/index.md [hunk 4]
```diff
@@ -1,159 +1,95 @@
...
+
+
+### Transition animation widgets
+
+*Transition animations*   NEED - description of transition widgets - what are they and why would you use transition instead of implicit or explicit animations.
```
Comment (InMatrix) [inline animations/index.md:34]: I saw the following in the staging site, but not here:
> Implicit animations are animations that are already programmed or styled within pre-existing constraints—all you need to do is add the animated object, and set some basic parameters such as the duration. You don’t need to write the animation code to add listeners or tickers for example, and you don’t need to tell the app when to start—the animation begins automatically when a property value changes. The AnimatedOpacity and AnimatedIcon widgets are two examples of implicit animations. 

This description is generally good. I'd ask an engineer to take a look to make sure it's accurate. Also, it won't hurt to mention AnimationController here is the key thing using a Transition widget gives you.

File: animations/index.md [hunk 5]
```diff
@@ -1,159 +1,95 @@
...
+### Explicit animation widgets
+
+ *Explicit animations* involve building customized animations using the `AnimationBuilder` widget that allows you to customize many of the animation elements that are pre-defined in the implicit and transition animation widgets. When you build explicit animations, you manually add listeners, tickers, and other elements for customization.  
+ <br>
+ For more information, see [Explicit animations](/animations/explicit_widgets).  
+
+## Choosing an animation widget
+Choosing an animation widget depends on the customization that you need for your animation and whether there is an existing widget available for reuse. The table below shows the customizable elements in each animation type.
+
+![Choosing an animation table](images/animation_table4.png)
```
Comment (InMatrix) [inline animations/index.md:49]: This table is useful, but it probably includes too much implementation detail that has not been introduced at this point. From the user's perspective, they would think about their choice in terms of how their animation needs to work. For example, is the animation triggered by user input, will the animation state influences other parts of the UI, or does the animation needs to simultaneously change several widgets or several properties of the same widgets?

Comment (InMatrix) [reply animations/index.md:49]: Suggestion: I'd ask an engineer on the framework team to identify a few animations in Flutter Gallery that are implemented using ImplicitAnimation widgets, Transition widgets, and AnimatedBuilder, respectively. And then, you can ask him to explain why some examples cannot be built using simpler solutions. This kind of decision process is very helpful to document for our users.

File: animations/index.md [hunk 6]
```diff
@@ -1,159 +1,95 @@
...
+
+### Animation and Motion widgets  
+For a list of Flutter animation widgets, see the following:
+* [Implicit animation widgets](/animations/implicit_widgets)  
+* [Transition animation widgets](/animations/transition_widgets)  
+* [Explicit animation widgets](/animations/explicit_widgets)  
+* [Animation and Motion Widgets Catalog](/widgets/animation/)  
+
+
+### Common animation patterns
```
Comment (InMatrix) [inline animations/index.md:82]: We have great content here. I would move this section up, perhaps before Flutter packages. I would also show animated illustrations next to each pattern listed below. We already have the assets in each subpage.

**Review by InMatrix — CHANGES_REQUESTED** [review]
I reviewed the index page only. This PR does include subpages, I'm not sure if you intend to ship them with the index page or not. If that's not your plan, links to pages that do not exist today need to be removed from the index page.

**Comment by Sfshaza** [comment]
Thanks for your review of the Animations index page, Tao! I still haven't
had a chance to look at it.

File: animations/index.md (hunk 1 above)
Comment (czakdesign) [reply animations/index.md:17]: ✔️

File: animations/index.md (hunk 2 above)
Comment (czakdesign) [reply animations/index.md:12]: ✔️

File: animations/index.md (hunk 3 above)
Comment (czakdesign) [reply animations/index.md:29]: i updated the index page to include more animations

File: animations/index.md (hunk 4 above)
Comment (czakdesign) [reply animations/index.md:34]: > reaching out to the developers for a tech review
> updated the content to include using the AnimationController w/transition widgets

File: animations/index.md (hunk 6 above)
Comment (czakdesign) [reply animations/index.md:82]: Great comment - although at this point there are already several animations on the page. I think the page is getting way to busy - and should probably be separated into some subpages - one page would be about the widget types - another would be about the animation patterns - another page would list out the additional resources such as the tech video pages, articles, tutorials, codelabs, etc.

my 2cents: segmenting information helps to provide information in small clips - pardon the animation pun - but adding all the information on the first page presents information that is not organized in a logical manner

**Comment by czakdesign** [comment]
changes are incorporated - although adding animations and reorganization will continue after technical review.

**Pull request closed without merging**